packages/webapp/pages/squads/[handle]/[token].tsx

```diff
@@ -179,21 +179,30 @@ const SquadReferral = ({ token, handle }: SquadReferralProps): ReactElement => {
             <BodyParagraph className="mt-2">@{source.handle}</BodyParagraph>
           </div>
           <Button
-            className="ml-auto btn-primary"
+            className="hidden tablet:flex ml-auto btn-primary"
             buttonSize="large"
             onClick={onJoinClick}
           >
             Join Squad
           </Button>
         </span>
-        <BodyParagraph className="mt-3 ml-[4.5rem]">
-          {source.description}
-        </BodyParagraph>
+        {source.description && (
+          <BodyParagraph className="mt-4 ml-[4.5rem]">
+            {source.description}
+          </BodyParagraph>
+        )}
+        <Button
+          className="flex tablet:hidden mt-4 ml-auto w-full btn-primary"
+          buttonSize="large"
+          onClick={onJoinClick}
+        >
+          Join Squad
+        </Button>
```

Comment:
Not sure if it's possible, but can't we put the bodyParagraph inside this element?
And then flex-reverse it on mobile?

Beats having two buttons that do the same in my opinion.

Reply:
Great suggestion, but not possible, unfortunately, the first button is inside a flex-row where the paragraph is outside of it and must be under those components.

Reply:
But looking at both mobile and desktop designs it's all inside that one box right?
Not quite sure why a flex solution would not work, seems to even simply push the button down that's all.

Reply:
Yes, but the box below, this must be contained, so if we add inside the description, it will be placed on the right side of the button, or if we will contain the two of them, the placement of the description will only be in the same column of the button.

Another option would be to contain the name, handle and description in the same column but then the description won't take the space on the right which is under the join button.

<img width="510" alt="image" src="https://user-images.githubusercontent.com/13744167/217733006-37b0ab4d-fe5b-4c93-a1ad-8db05f3e6bf8.png">

Reply:
Oh right, didn't take the description into account.
I don't have a issue with it, but never like multiple buttons, danger lies in changing behaviour/text/etc.

Reply:
I agree, me too, just had no other choice 😭

Reply:
Suggestion for that: Can we maybe make it 1 shared component on top of the file? or outside the component so we limit the risk.

Reply:
Good suggestion! Let me add that, probably good to have test cases as well to ensure when they are both displayed.

Reply:
Made the function and applied tests as well.